README.md

```
@@ -49,31 +55,67 @@ data/plugins/
     └── schema.json
 ```
 
-重启 Kira 后插件会自动被发现并加载。可在 WebUI 的插件管理页面中启用/禁用及调整配置。
+**方式二：内置插件**
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)